flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -636,8 +652,6 @@ private static void start(YarnConfiguration conf, String principal, String keyta
 		Assert.assertNotNull("Flink uberjar not found", flinkUberjar);
 		String flinkDistRootDir = flinkUberjar.getParentFile().getParent();
 		flinkLibFolder = flinkUberjar.getParentFile(); // the uberjar is located in lib/
-		// the hadoop jar was copied into the target/shaded-hadoop directory during the build
```

Comment:
the pom should contain something to remove

Reply:
It does: https://github.com/apache/flink/pull/11983/files#diff-b2b20de90abbd8256722ee4019e44ea2L408